mldsa/src/poly_kl.h

```diff
@@ -9,7 +9,7 @@
 #include "common.h"
 #include "poly.h"
 
-#define mld_poly_decompose MLD_NAMESPACE(poly_decompose)
+#define mld_poly_decompose MLD_NAMESPACE_KL(poly_decompose)
```

Comment:
Ok, so `poly_decompose`, `poly_{make, use}_hint` do depend on the security level and should always have been KL-namespaced. But `poly_chknorm` is generic, or?

Reply:
Right.